src/content/blog/2024-02-21-eslint-2023-year-review.md

```diff
@@ -0,0 +1,96 @@
+
+## Other changes
+
+* [Deprecation of formatting rules](https://eslint.org/blog/2023/10/deprecating-formatting-rules/) - in a significant shift from ESLint's roots, we decided to officially deprecate formatting rules
+* [Changes to rule API](https://eslint.org/blog/2023/09/preparing-custom-rules-eslint-v9/) - for those who write their own rules, we announced important changes to the rule API that will prepare ESLint for our next step: [language plugins](https://github.com/eslint/rfcs/blob/main/designs/2022-languages/README.md).
+
+## Updates to support the latest ECMAScript features
+
+The team spends much of the year anticipating new ECMAScript features and updating ESLint accordingly. 2023 was an outlier in that most of the changes weren't syntax-related, and so there wasn't a lot of additional work. The most significant syntax change, [hashbang comments](https://github.com/tc39/proposal-hashbang), is something that ESLint already handled to support Node.js.
+
+## Income review
+
+During 2023, ESLint received $179,601.53 USD from all of our income sources. You can see the breakdown of our income sources in the following table.
+
+| **Source** | **Amount** |
+|-----------|-------------|
+| Open Collective | $117,336.30 |
+| Tidelift | $30,077.84 |
+| GitHub Sponsors | $25,190.89 |
+| Carbon Ads | $6,380.89 |
+| Thanks.dev | $524.61 |
+| Stackaid.us | $91.00 |
+| **Total** | **$179,601.53** |
+
+As usual, most of our donations came through Open Collective. Tidelift (slight up from 2022) surpassed GitHub Sponsors donations (slightly down from 2022) in 2023. Notably, our website ad revenue dropped roughly 50% from 2022, following an overall softening of the display ad business. We'll be looking at options to improve our website ad revenue in 2024.
+
+The newer sponsoring options, Thanks.dev and Stackaid.us, didn't provide much income but are both interesting ideas that we'll keep participating in.
+
+We are grateful to the companies and individuals who generously donated to support ESLint. We know that there has been a lot of turbulence in the tech industry in the past year, and we really appreciate everyone who still decided to sponsor ESLint.
+
+## Expenses review
+
+In 2023, we spent $182,245.97 USD on the ESLint project as a whole. This includes regular maintenance and development as well as other costs related to the project. See the table below for an overview.
+
+| **Category** | **Amount** |
+|-----------|-------------|
+| Maintenance and Development | $66,910.37 |
+| TSC Stipend (Tidelift) | $30,077.84 |
+| Triage and Support | $10,307.78 |
+| Ecosystem/Dependencies | $24,100.00 |
+| Community Management | $15,599.84 |
+| Health Insurance | $13,447.41 |
+| Open Source Collective Fee | $8,960.38 |
+| Technical Writing | $8,155.00 |
+| Contributor Pool | $4,500.00 |
+| Google Workspace | $187.35 |
+| **Total** | **$182,245.97** |
```

Comment:
```suggestion
In 2023, we spent $185,646.98 USD on the ESLint project as a whole. This includes regular maintenance and development as well as other costs related to the project. See the table below for an overview.

| **Category** | **Amount** |
|-----------|-------------|
| Maintenance and Development | $66,910.37 |
| TSC Stipend (Tidelift) | $30,077.84 |
| Triage and Support | $10,307.78 |
| Ecosystem/Dependencies | $24,400.00 |
| Community Management | $15,599.84 |
| Health Insurance | $13,447.41 |
| Open Source Collective Fee | $9,206.88 |
| Payment Processor Fee | $2,854.51 |
| Technical Writing | $8,155.00 |
| Contributor Pool | $4,500.00 |
| Google Workspace | $187.35 |
| **Total** | **$185,646.98** |
```

Ecosystem/Dependencies $24,100.00 -> $24,400.00
Open Source Collective Fee $8,960.38 -> $9,206.88

Plus there's also a not-so-insignificant amount that goes on payment processor fees ($2,253.39 on income + $601.12 on expenses = $2,854.51).

Reply:
> Plus there's also a not-so-insignificant amount that goes on payment processor fees ($2,253.39 on income + $601.12 on expenses = $2,854.51).

Ah interesting. Apparently these charges don't get exported by default. I'll re-run the spreadsheet with the updated transactions.

Reply:
Okay got it. The payment processor fees show up as separate transactions on opencollective.com, but in the exported data it's a separate column. 👍